README.md

```diff
@@ -56,8 +56,8 @@ to Miri and fix it there.
 
 Licensed under either of
 
-* Apache License, Version 2.0 ([LICENSE-APACHE](LICENSE-APACHE)
-* MIT license ([LICENSE-MIT](LICENSE-MIT)
+- Apache License, Version 2.0 ([LICENSE-APACHE](LICENSE-APACHE)
+- MIT license ([LICENSE-MIT](LICENSE-MIT)
```

Comment:
Why did you change the bullet style?

Reply:
Both because `prettier` has that as the preferred style, and because stackoverflow told me that GitHub actions needs an additional commit after the workflow file is created. It turned out my issue was with my yaml formatting, which for some reason invalid syntax on the second item indented at the same level.

Reply:
Can you move this to a separate PR?